File: src/main/java/com/adjorno/billib/rest/db/GlobalRankTrack.java
package com.adjorno.billib.rest.db;

import javax.persistence.*;

@Entity
@Table(name = "GLOBAL_RANK_TRACK")
public class GlobalRankTrack {
    @Id
    @Column(name = "RANK")
    @GeneratedValue(strategy = GenerationType.AUTO)
    private Long mRank;

    @Column(name = "TRACK_ID")
    private Long mTrackId;

    public Long getRank() {
        return mRank;
    }

    public void setRank(Long rank) {
        mRank = rank;
    }

    public Long getTrackId() {
        return mTrackId;
    }

    public void setTrackId(Long trackId) {
        mTrackId = trackId;
    }
}

File: src/main/java/com/adjorno/billib/rest/db/GlobalRankTrack.kt
package com.adjorno.billib.rest.db

import jakarta.persistence.Column
import jakarta.persistence.Entity
import jakarta.persistence.GeneratedValue
import jakarta.persistence.GenerationType
import jakarta.persistence.Id
import jakarta.persistence.Table

@Entity
@Table(name = "GLOBAL_RANK_TRACK")
data class GlobalRankTrack(
    @Id
    @Column(name = "_RANK")
    @GeneratedValue(strategy = GenerationType.IDENTITY)
    var rank: Long? = null,

    @Column(name = "TRACK_ID")
    var trackId: Long? = null
)
